status-im/status-react
# Switched to newer react-native-desktop with updated dependencies

fixes vulnerable desktop dependenccies

### Summary:

Switched to newer react-native-desktop with updated `yeoman-generator`

### Review notes (optional):
@mandrigin, could you please check that now we have correct dependencies?

### Testing notes (optional):
No code was affected. Also updated `yeoman-generator` doesn't participate in app work. `react-native-desktop` uses it to generate new projects.
So only basic checks on app needed.

#### Platforms (optional)

- macOS
- Linux
- Windows


#### Areas that maybe impacted (optional)
**Functional**
- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

**Non-functional**
- battery performance
- CPU performance / speed of the app
- network consumption


### Steps to test:
- Open Status
- Make sure it runs well



status: ready
